### PR TITLE
Added more wood and potion types.

### PR DESCRIPTION
**Pull request by ghost**
Added more Wood (Log) and Potion types.

Removed unnecessary quart pillar blocks.

### CODE REVIEW

File: Lib/src/main/java/me/desht/dhutils/ItemNames.java [hunk 1]
```diff
@@ -438,15 +442,21 @@
             .put("373:16418", "Swiftness Splash II (1:07)")
             .put("373:16420", "Poison Splash II (0:16)")
             .put("373:16421", "Healing Splash II")
+            .put("373:16422", "Night Vision Splash (2:15)")
             .put("373:16425", "Strength Splash II (1:07)")
             .put("373:16428", "Harming Splash II")
+            .put("373:16429", "Water Breathing Splash (2:15)")
+            .put("373:16430 ", "Invisibility Splash (2:15)")
```
Comment (desht) [inline Lib/src/main/java/me/desht/dhutils/ItemNames.java]: Trailing space here

**Comment by desht** [comment]
Did you also mention new wood types?  I don't see anything in the PR related to that.

**Comment by ghost** [comment]
I reverted the change as I realised the wood types (Dark Oak, Acacia) I added don't have the same ID as Oak/Spruce/Birch/Jungle wood types, and you had already included them in your original version.

https://github.com/DoctorDark/dhutils/commit/7ee237e9ddcf2da314e249837236a80d83b97012
Acacia Wood should be 162 for example, not 17:4, and your original version also has 162 added, same goes for the saplings

**Comment by desht** [comment]
Pulled, thanks!

**Pull request merged**